server/.env.sample

```diff
@@ -17,4 +17,5 @@ EXPRESS_SESSION_FILESTORE_PATH=/tmp/express-sessions
 EXPRESS_PRELOADED_STATE_FILE=/tmp/revealState.json
 
 EXPRESS_SESSION_LOGIN_URL=/login
-FRONTEND_OPENSRP_CALLBACK_URL=http://localhost:3000/fe/oauth/callback/opensrp
\ No newline at end of file
+FRONTEND_OPENSRP_CALLBACK_URL=http://localhost:3000/fe/oauth/callback/opensrp
+FRONTEND_LOGIN_URL=/fe/login
```

Comment:
Can we append `EXPRESS_` to these please?

Reply:
yeap